collections/_drafts/agent-forwarding.md

```diff
@@ -1,13 +1,11 @@
 ---
 title: OpenINF Next-Gen Guidance on Agent Forwarding
 category: contributing
-authors: @OpenINF/wg-a-team
```

Comment:
i had removed this originally bc i thought it was what broke the document, but not actually sure

Reply:
i will go see if there is a plugin about proper attribution

Reply:
> A plugin to seamlessly support multiple authors with paginated posts inside a jekyll powered publication blog. Extends jekyll-paginate-v2 for Autopages and Pagination.
> 🔗 https://rubygems.org/gems/jekyll-auto-authors